### PR TITLE
[Font Awesome] Fix for inline SVGs breaking font family declaration

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR fixes an issue with Font Awesome's SVGs being declared inline, breaking the font family definition. It fixes this by adding a limit on the size of an SVG that should be inlined in Webpack.

Requires publish of https://github.com/department-of-veterans-affairs/design-system/pull/227.

## Testing done
- Tested site locally

## Screenshots
I noticed this in the compiled CSS -

image

Per @aubreyarcangel's comment here, https://github.com/department-of-veterans-affairs/design-system/pull/227#issuecomment-457261098, this SVG definition was the issue. By adding an extra parameter to the Webpack SVG loader, we can make sure that the FA SVGs aren't declared inline, which apparently doesn't work.

## Hubs
image

## Modals

image

image

image

### Alert banners
image

image

image

image

### Social Icons
image

## Acceptance criteria
- [ ] Font Awesome works properly

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/iconography.scss [hunk 1]
```diff
@@ -12,15 +12,16 @@
 @mixin make-hub($hub-name,
   $icon,
   $color,
+  $icon-weight: 400,
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/iconography.scss]: 400/900  is the distinction between switching from FA Regular to FA Solid.

File: src/site/components/merger-social.html [hunk 2]
```diff
@@ -40,7 +44,11 @@ <h4>{{ subs.subhead }}</h4>
                     {% endif %}
                     >
                     {% if link.icon != empty %}
-                      <i class="fa fab {{link.icon}}"></i>
+                      {% if is_social_media_links %}
+                        <i class="fab {{link.icon}}"></i>
+                      {% else %}
+                        <i class="fas {{link.icon}}"></i>
+                      {% endif %}
```
Comment (ncksllvn) [inline src/site/components/merger-social.html]: I don't like this, but this template will be deprecated when the Drupal template comes along.

File: src/platform/site-wide/sass/iconography.scss [hunk 3]
```diff
@@ -126,6 +132,7 @@
   $hub-name: "life-insurance",
   $icon: $fa-var-shield-alt,
   $color: $green-warm-50,
+  $icon-weight: 900,
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/iconography.scss]: `900` indicated to use the solid icon set of FA Free, while 400 is the regular set.

**Comment by ncksllvn** [comment]
This PR is using the alpha version of https://github.com/department-of-veterans-affairs/design-system/pull/227. If this looks okay, I'm going to merge 227, publish an official version, then update this PR to use that instead of the alpha.

File: config/webpack.config.js [hunk 4]
```diff
@@ -129,7 +129,7 @@ const configGenerator = (buildOptions, apps) => {
         {
           test: /\.svg/,
           use: {
-            loader: 'svg-url-loader',
+            loader: 'svg-url-loader?limit=1024',
```
Comment (ncksllvn) [inline config/webpack.config.js]: This was the main culprit.

File: src/applications/claims-status/components/MailOrFax.jsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ export default function MailOrFax({ onClose }) {
         aria-label="Close this modal"
         onClick={onClose}
       >
-        <i className="far fa-times-circle" />
+        <i className="fas fa-times-circle" />
```
Comment (ncksllvn) [inline src/applications/claims-status/components/MailOrFax.jsx]: I think there was a mixup here. According to the [design channel](https://dsva.slack.com/archives/C0NGDDXME/p1548260326063800), we want to be using the close button in the screenshot below -

image

This is as it is in the site now. However, this is the [solid times-circle](https://fontawesome.com/icons/times-circle?style=solid), not the regular (`far`). I think the `far` class was falling back to `fas`, yielding the correct result, but not the right implementation.

Comment (ncksllvn) [reply src/applications/claims-status/components/MailOrFax.jsx]: This is how it is now -

image

File: src/platform/site-wide/sass/iconography.scss [hunk 6]
```diff
@@ -36,9 +36,10 @@
     text-rendering: auto;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+    font-weight: 900;
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/iconography.scss]: `font-weight: 900` is what triggers using the solid icon set of FA 5 Free.

**Comment by ncksllvn** [comment]
@amycesal @emilywaggoner Can you all use the Heroku instance here to confirm the site looks okay? Fixed some stuff in the design system, and want to make sure no visual bugs as a result. We had some clashes of solid vs. regular Font Awesome icons and I tried correcting them.

**Review by jbalboni — APPROVED** [review]

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 7]
```diff
@@ -401,7 +401,7 @@
       text-decoration: underline;
 
       &:after {
-        font: normal normal normal 1em FontAwesome;
+        font: normal normal normal 1em 'Font Awesome 5 Free';
         content: "\F078";
```
Comment (aubreyarcangel) [inline src/applications/facility-locator/sass/facility-locator.scss]: This doesn't show up for me on my local build 
screen shot 2019-01-25 at 3 16 22 pm

**Pull request merged**